pkg/genlib/generator_with_text_template_test.go

```diff
@@ -755,6 +755,64 @@ func Test_FieldIPWithTextTemplate(t *testing.T) {
 	}
 }
 
+func Test_FieldLongCounterResetAfterN5WithTextTemplate(t *testing.T) {
```

Comment:
Add cases where the counter is incrementing properly between multiple resets to consider the case in my previous comment.